inspirehep/alembic/7be4c8b5c5e8_inspirehep_new_migrations.py

```diff
@@ -0,0 +1,323 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""Inspirehep initial revision of migrations
+which makes db identical like in inspire-next"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "7be4c8b5c5e8"
+down_revision = None
+branch_labels = ("inspirehep_new",)
+depends_on = "dbdbc1b19cf2"
+
+
+def upgrade():
+    # """Upgrade database."""
```

Comment:
why? ☺️

Reply:
It's only for allowing to run tests from inspire-next.  (we were talking about it many times ;) )
This migration will never be executed on production or QA, it's for local environment only.

Reply:
But still :/ I don't understand why is on``inspirehep``? ``inspirehep`` should know **nothing** about ``inspire-next``, right? Why we cannot just run inspirehep migrations and then ``inspire-next`` migrations  to add the missing tables?

Reply:
We were talking about this already. You cannot run migrations from inspirehep and inspire-next on the same db, as both projects are using the same modules.

Reply:
The following scenario will not work? 

```bash
$ (inspirehep) -> poetry run inspirehep alembic upgrade
```

```shell
$ (inspire-next) -> inspirehep alembic upgrade
```

Reply:
Of course it won't. All tables for invenio-records, invenio-files will be already in db after first command, and alembic from second one will scream and refuse to migrate.

Reply:
what about ignore tables?